### DISCUSSION TITLE
Curiously, is it secure to publish admin password  in repository ?

### DISCUSSION THREAD

**Discussion by Marvin-Bai** [RocketChat/Rocket.Chat.ReactNative#2856] [Q&A] [answered]
I can see the admin password and server url in e2e/data, to my knowledge, it is insecure to push such account and  password in public repository, while I don't know whether it is confidential.

**Comment by Marvin-Bai** [comment, marked as answer]
Sorry, I just checked it, I see it's for example :)

**Reply by diegolmello** [reply]
Yeah, it's been used for our e2e tests on CI and it's not really the admin of that server.
The user has just a permission to add other users to the server to setup tests env.
A similar permission is given to any user on that server to create its own user.

**Reply by Marvin-Bai** [reply]
Got it, Thks. 😁
